README.md

```diff
@@ -35,6 +35,8 @@ ecAudit integrates with Apache Cassandra using its existing plug-in points.
 Official releases of ecAudit can be downloaded from Maven Central.
 Get the ecAudit flavor for your Cassandra version.
 
+[![ecAudit for Cassandra 4.1.<latest>](https://img.shields.io/maven-central/v/com.ericsson.bss.cassandra.ecaudit/ecaudit_c4.1.svg?label=ecAudit%20for%20Cassandra%204.1.<latest>)](https://search.maven.org/search?q=g:%22com.ericsson.bss.cassandra.ecaudit%22%20AND%20a:%22ecaudit_c4.1%22)
```

Comment:
This badge doesn't work but i guess it will once it it's released. So ignore this comment if that's intended.

Reply:
That's the intention.